bin/picca_deltas.py

```diff
@@ -613,18 +613,27 @@ def main(cmdargs):
 
     ### Mask BALs
     if not args.bal_catalog is None:
-        userprint("INFO: Adding BALs found in BAL catalog")
-        bal_cat = bal_tools.read_bal(args.bal_catalog)
+        userprint("INFO: Masking BALs from BAL catalog")
+        bal_cat = bal_tools.read_bal(args.bal_catalog,'EBOSS')
```

Comment:
is this right? wouldn't this only allow using EBOSS cats? Or is reading DESI cats just unneccessary because DESI BALs are always fed into the QSO cat?

Reply:
Reading the bal_tools.read_bal the survey is set to 'DESI' by default, but with this it is always called usin  as 'EBOSS'. To fix this, I would add a parsing option (with 'EBOSS' as default if you want) and then pass `args.survey` to this function

Reply:
Is args.survey already an argument for picca_deltas? I don't see it available.

The EBOSS quasar catalogs do not contain the BAL information needed for masking, so separate BAL catalogs were made. As a result, in EBOSS, I need a way to match the quasars in both catalogs. I have been using 'THINGID' for simplicity, but this results in a naming clash with DESI, where we start using 'TARGETID'. I'm currently not aware of any easy way to switch between the naming conventions of the two surveys, but if there is I'd be happy to incorporate it.

 For DESI, we're planning on having BAL information added to the quasar catalogs as an afterburner, so no additional BAL catalog is needed.

Reply:
No, it is not an argument yet. My suggestion is to create one so that we can choose the survey we want to use (currently we are always running in eBOSS mode).

I'm not sure what you mean in the second and third paragraphs.